x-pack/plugins/security_solution/public/detections/components/rules/step_rule_actions/index.tsx

```diff
@@ -114,16 +116,18 @@ const StepRuleActionsComponent: FC<StepRuleActionsProps> = ({
     }
   }, [form, setForm]);
 
-  const updateThrottle = useCallback((throttle) => setMyStepData({ ...myStepData, throttle }), [
-    myStepData,
-    setMyStepData,
-  ]);
+  const updateThrottle = useCallback(
```

Comment:
We might need to zoom as we never use `getFields()` in our forms, and I see that you do in need it in multiple places and manually set the value `setValue()`

Reply:
Yep, this one was not needed (https://github.com/elastic/kibana/pull/76138/commits/f36baf564812b2377e8cc453c546b49661688f8f), and the local state was later replaced with `useFormData` 👍